data/tlg1595/tlg370/__cts__.xml

```diff
@@ -2,6 +2,6 @@
     <ti:title xml:lang="lat">Volumina Rhetorica</ti:title>
     <ti:edition urn="urn:cts:greekLit:tlg1595.tlg370.1st1K-grc1" workUrn="urn:cts:greekLit:tlg1595.tlg370">
         <ti:label xml:lang="grc">ΠΕΡΙ ΡΗΤΟΡΙΚΗΣ</ti:label>
-        <ti:description xml:lang="mul">Philodemus, Volumina Rhetorica, Sudhaus, Teubner, 1895</ti:description>
+        <ti:description xml:lang="mul">Philodemus. Volumina Rhetorica, Volume 3: Supplementum. Sudhaus, Siegfried. Leipzig: Teubner, 1895.</ti:description>
   </ti:edition>
```

Comment:
Is this the editor?
`Sudhaus, Siegfried, editor.`

Reply:
Knew I missed something, yes it it.